tests/Aspire.Hosting.SqlServer.Tests/SqlServerFunctionalTests.cs

```diff
@@ -68,18 +70,18 @@ public async Task VerifySqlServerResource()
         using var builder = TestDistributedApplicationBuilder.Create(o => { }, testOutputHelper);
 
         var sqlserver = builder.AddSqlServer("sqlserver");
-        var tempDb = sqlserver.AddDatabase("tempdb");
+        var newDb = sqlserver.AddDatabase(databaseName);
```

Comment:
Not using `tempdb` updates this test to ensure the database is created. `tempdb` was used since it's a default one in sql server.